Sources/Core/Renderer/Mesh/FluidMeshBuilder.swift

```diff
@@ -107,9 +107,11 @@ struct FluidMeshBuilder { // TODO: Make fluid meshes look more like they do in v
     tint: Vec3f
   ) {
     let basePosition = position.relativeToChunkSection.floatVector + Vec3f(0.5, 0, 0.5)
+    var cullingFluid = cullingNeighbours
```

Comment:
The variable name `cullingFluid` doesn't really make sense for a Set to me. Maybe just do `var cullingNeighbours = cullingNeighbours` instead